<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HouseWindow</class>
 <widget class="QMainWindow" name="HouseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
border-image: url(:/image/room/房间主背景.jpg);
}</string>
   </property>
   <widget class="QWidget" name="ledWidgt_1" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closeled_ico.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="ledwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closeled_ico.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="funwidget" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closefan_ico.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="doorwidget" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closedoor_ico.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ledPushButton_1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ledPushButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="funPushButton_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="doorPushButton_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
